src/provisioningd.cpp

```diff
@@ -232,20 +291,24 @@ int main(int argc, const char* argv[])
                                           std::ref(bmcResponder), sport),
                           net::detached);
         });
-        controller.setCheckPeerHandler([&]() {
-            LOG_INFO("Checking peer BMC connection");
-            net::co_spawn(io_context,
-                          std::bind_front(tryConnect, std::ref(io_context), ip,
-                                          rport, std::ref(controller)),
-                          net::detached);
-        });
 
         DbusSignalWatcher<bool>::watch(
             io_context, conn,
             std::bind_front(onSpdmStateChange, std::ref(io_context), ip, sport,
                             std::ref(controller), std::ref(bmcResponder),
                             rport),
             SPDM_RES_INTF, SPDM_RES_SIGNAL);
+        DbusPropertyWatcher<std::string>::watch(
+            io_context, conn,
+            std::bind_front(onNeighbhorFound, std::ref(io_context),
+                            std::ref(controller), rport),
+            std::format(LLDP_REC_PATH, "eth1"), LLDP_INTF, LLDP_PROP);
```

Comment:
I think D-bus watch must look for object added signal right?

Reply:
I suggest to use InterfaceAdded signal uniformly across all applications (spdm, rsync, provisioningd)